deploy/mac/MacOSXBundleInfo.plist.in

```diff
@@ -9,7 +9,7 @@
     <key>CFBundleGetInfoString</key>
     <string>${MACOSX_BUNDLE_INFO_STRING}</string>
     <key>CFBundleIconFile</key>
-    <string>${MACOSX_BUNDLE_ICON_FILE}</string>
```

Comment:
I think this will be fixed in #11362, when I set the MACOSX_BUNDLE_ICON_FILE property. I didn't realize it wasn't supposed to have the path but just the file name.

Reply:
I tried fixing it that way and it didn't work. I got tired of fighting with it and gave up trying to do it the right way.